src/Compilers/CSharp/Test/CommandLine/CSharpCommandLineTest.csproj

```diff
@@ -79,6 +77,18 @@
     <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="xunit.abstractions, Version=$(xUnitAbstractionsAssemblyVersion), Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
```

Comment:
I would suggest leaving the version info out of the Identity and getting rid of the $(*AssemblyVersion) properties, but I don't feel strongly about it.